google-ads-googleads.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.5.0'
   s.summary = 'Google client library for the Google Ads API'
 
-  s.add_dependency 'gapic-common', '~> 0.20'
+  s.add_dependency 'gapic-common', '~> 0.21.1'
```

Comment:
change it to `['>= 0.21.1', '< 1.0']`

The `~>` operator isn't going to do what we want here, and will be overly restrictive

Reply:
I addressed the comment. but I have 2 identical commit messages. Should I squash them?